src/mocks/characterMocs.ts

```diff
@@ -0,0 +1,299 @@
+import { RowCharacter } from '../api/starWars-types'
```

Comment:
Might be better to put this under API folder as it's the closest place that the original value(the type definition and the logic to retrieve this data in this case).